src/lib.js

```diff
@@ -1920,10 +1920,28 @@ export const getStats = async (
       output.slayer_coins_spent.total = output.slayer_coins_spent.total || 0;
     }
 
+    if (profile.uuid == "b44d2d5272dc49c28185b2d6a158d80a") {
```

Comment:
I don't like that the slim easter egg is in lib.js because it could potentially affect other things. it would be better to keep it in stats.ejs